.github/workflows/build.yml

```diff
@@ -0,0 +1,118 @@
+name: Build
+on:
+  pull_request:
+    branches:
+      - main
+      
+jobs:
+    build-ali-cli:
+        name: Build Only (No Release)
+        runs-on: [ubuntu-latest]
+        steps:
+        - uses: actions/checkout@v4
```

Comment:
Shouldn't we use `checkout@v5` everywhere?

Reply:
Adding a `.github/dependabot.yml` config this allow this to be auto-updated.

Ex: https://github.com/cloudfoundry/bosh-agent/blob/main/.github/dependabot.yml